book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialise(title, _author)
+    @title = title
+    @author = title
```

Comment:
- [ ] May you mean `@author = author` ? Please 🙏🏽 fix your code and review the code to create books to avoid errors.